src/themes.lisp

```diff
@@ -26,9 +26,25 @@ function that takes a DOCUMENT and returns NIL or a STRING for template insertio
   (or (find-package (format nil "~:@(coleslaw.theme.~A~)" name))
       (error 'theme-does-not-exist :theme name)))
 
-(defun theme-fn (name &optional (package (theme *config*)))
+(defun get-djula-theme (name)
+  (symbol-value (intern (string-upcase (format nil "*djula-~A-template*" (string name))))))
+
+(defun theme-fn (name &optional
+                        (package (theme *config*))
+                        (templating-engine *templating-engine*))
```

Comment:
Wouldn't `template-engine` be more in line with the rest of the wording instead?